sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache.cpp

```diff
@@ -10,6 +10,8 @@
 // RUN: %{build} -o %t.out -ffp-model=precise
 // RUN: %{run} %t.out
 
+// XFAIL: gpu-intel-dg2
+
```

Comment:
Same as above:
Adding XFAIL will make the code fail silently for the other supported combination.